### ISSUE TITLE
hieronymous

### ISSUE THREAD

**Issue by eroux** [biblissima/collatinus#28]
Hieronymous (et Hierusalem, etc.) est un cas assez particulier car ils est écrit "Hieronymus" dans les graphies ramistes (noté _Hĭĕrōnўmus_ dans le Gaffiot et _Hĭĕrōnymus_ dans le L&S), et pourtant, au en latin liturgiques, il est prononcé avec un premier i consonne, et donc devrait théoriquement être écrit "Hjeronymus" (qui est assez moche). Pedecerto donne un (seul) exemple de Hieronymus avec un i voyelle et 0 avec un i consonne, donc on peut penser que les deux existent. La question est : comment faire pour qu'une personne que seul le latin ecclésiastique intéresse puisse ne trouver que Hjeronymus et jamais Hieronymus ?

**Comment by ycollatin** [comment]
Le 12 mai 2016, à 07h31, Elie Roux écrivit :

> Hieronymous (et Hierusalem, etc.) est un cas assez
> particulier car ils est écrit "Hieronymus" dans les
> graphies ramistes (noté _Hĭĕrōnўmus_ dans le Gaffiot et
> _Hĭĕrōnymus_ dans le L&S), et pourtant, au en latin
> liturgiques, il est prononcé avec un premier i consonne,
> et donc devrait théoriquement être écrit "Hjeronymus" (qui
> est assez moche). Pedecerto donne un (seul) exemple de
> Hieronymus avec un i voyelle et 0 avec un i consonne, donc
> on peut penser que les deux existent. La question est :
> comment faire pour qu'une personne que seul le latin
> ecclésiastique intéresse puisse ne trouver que Hjeronymus
> et jamais Hieronymus ?

```
Philippe te répondra mieux que moi, il sent beaucoup
```

mieux que moi la prosodie, et il est aidé par sa
connaissance de l'italien. Mais il me semble qu'il sera
difficile de se sortir de ce i consonne. Dès le début de
l'Énéide, on lit :

Arma uirumque cano, Troiae qui primus ab oris
Italiam, fato profugus, Lauiniaque ueni

Or, le deuxième i de /Lauiniaque/ est en fait un j, mais on ne
peut pas décemment écrire /Lavinjaque/. De même pour /etiam/ :
c'est et + jam, mais on ne l'écrit jamais /etjam/. Pour le
v, nous avons 'ụ', qu'il faut manipuler pour l'afficher 'u'.
Peut-être faudra-t-il faire la même chose pour le i.

S'il apparaît que les particularités du latin liturgique
sont trop nombreuses, nous aurons intérêt, je pense, à
discuter des moyens de rendre modulaire notre système de
données, en faisant des paquets spécialisés, qu'on
ajouterait optionnellement au tronc commun. Ce n'est pas
pour tout de suite.

Hors-sujet : J'ai continué de corriger la nouvelle version
de lemmes.la, et j'ai fait le bilan 
- de ce qui n'était pas traduit dans lemmes.fr, 
- de ce que lemmes.fr traduit, mais n'est pas dans
  lemmes.la. C'est moins catastrophique que ce que je
  craignais.

I. Uniquement dans lemmes.la :
Asina
Chĕrusci
cŏtidianus
cŏturnix
Cy̆clops
cy̆cnus
elegea
erĭlis
excĭtus
fortuĭtus
hisce
illecebrosius
inpeditius
iris2
Jesus
Ligones
lĭquidus
lotus
Lŏcusta
molitor2
pŏlypus
Prŏserpina
quoquo
rĕliquiae
Seplăsia
serŏ3
sŏrex
supremus
Sy̆chaeus
sy̆cophantia
ubĭcumque
Vedius2
virus

II. uniquement dans lemmes.fr :

ab
alipta
asina
Cherusci
concido2
cotidianus
coturnix
Cyclops
cycnus
dextera
elegia
erilis
excitus
exilio
exilium
extinguo
exto
extruo
fortuitus
indu
intelligo
Iris2
jurisjurandum
levitas2
liquidus
Locusta
memento
Menippaeus
obstipesco
optumus
Pandora
plures
polypus
post2
potesse
prae2
praeter2
Proserpina
pulcer
quamquam
reliquiae
Seplasia
sero3
sorex
Sychaeus
sycophantia

## ubicumque

```
                    Yves
```

**Comment by eroux** [comment]
Effectivement, s'il y a des i consonnes qui ne sont pas notés j en graphie ramiste, il faut pouvoir les indiquer d'une façon ou d'une autre ! Cela pose plusieurs questions : 
- comment distinguer un i consonne qui doit être écrit `i` et un i consonne qui doit être écrit `j` ?
- comment indiquer ces i dans les lemmes ? (je propose un i sans point : ı)
- comment les indiquer à l'utilisateur, dans l'onglet lexique par exemple ? mettre ĭ ne peut que porter à confusion... pourquoi pas le même i sans point ?

pour ce qui est d'un module spécifique au latin liturgique, pour l'instant il n'y en a pas l'utilité, tout ce que repère n'a pas l'air spécifique... La seule fonctionnalité à laquelle je pense serait une option du lemmatiseur pour privilégier les formes avec un j s'il existe aussi une forme avec un i (par exemple priviégier Jācōb plutôt que Iācōb), mais ça peut se faire avec un lexique personnalisé contenant des indices de rareté...

**Comment by PhVerkerk** [comment]
Je n'ai pas d'opinion sur Hieronimus. Tous les dicos indiquent un vrai i bref. Mais il y a peut-être une différence entre le philosophe de Cicéron, le roi de Syracuse de Tite Live ou Jérôme que nous connaissons.
Le i sans point me va bien. On peut toujours prévoir une option pour cacher/montrer le i sans point ou le u exponctué. Le défaut du i sans point est qu'il n'existe pas en majuscule, mais un i consonne en début de mot se note peut-être toujours j.

Privilégier Jacob par rapport à Iacob se ferait bien avec un indice de rareté ou de fréquence.

Ph. 

Le Jeudi 12 Mai 2016 18:50 CEST, Elie Roux notifications@github.com a écrit:

**Comment by eroux** [comment]
Ok, je pense qu'on peut partir là-dessus alors... une fois que ce sera fait il suffira d'ajouter `Hıĕrōnўmus` dans le lexique et nous pourrons fermer le ticket, j'en ouvre un pour les indices de rareté/fréquence sur les lemmes et un sur le `ı`
